pages/common/redshift.md

```diff
@@ -0,0 +1,27 @@
+# redshift
+
+> Adjust the color temperature of your screen according to your surroundings.
+
+- Turn on Redshift with 5700K temperature during day and 3600K at night:
+
+`redshift -t {{5700}}:{{3600}}`
+
+- Turn on Redshift with a manually-specified custom location:
+
+`redshift -l {{latitude}}:{{longitude}}`
+
+- Turn on Redshift with 70% screen brightness during day and 40% brightness at night:
+
+`redshift -b {{0.7}}:{{0.4}}`
+
+- Turn on Redshift with custom gamma levels (between 0 and 1):
+
+`redshift -g {{red}}:{{green}}:{{blue}}`
+
+- Turn on Redshift with temperature transitions disabled:
+
+`redshift -r`
```

Comment:
The `-r` flag just makes it so that Redshift changes the color temperature instantly, instead of gradually fading to the proper temperature. It's not a terribly important flag; it could probably be dropped from this page, actually.